app/controllers/dashboard_controller.rb

```diff
@@ -442,10 +442,10 @@ def authenticate(require_api_token = false)
     }
 
     if params[:user_name].blank? && params[:user_password].blank? &&
-      request.headers["X-Remote-User"].blank? &&
-      get_vmdb_config[:authentication][:mode] == "httpd" &&
-      get_vmdb_config[:authentication][:sso_enabled] &&
-      params[:action] == "authenticate"
+       request.headers["X-Remote-User"].blank? && 
+       ::Settings.authentication.mode == "httpd" &&
+       ::Settings.authentication.sso_enabled &&
+       params[:action] == "authenticate"
```

Comment:
Ohhhh because it is attached to the && from the line before.  This is fine then.